Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml

```diff
@@ -455,3 +472,5 @@
     items:
     - BikeHorn
     - ClownRecorder
+    - CrayonRainbow
```

Comment:
feed or use up the crayon for esword slot???

Reply:
ah yea i forgot it disappears when it gets eat